docs/compute-engine/quickstart_android.md

```diff
@@ -1,19 +1,172 @@
+[LCE package hosted at Bintray](https://bintray.com/plumeraihq/larq-compute-engine).
+Modify the `build.gradle` dependencies in the anroid project to include the
+`lce-lite` target:
+
+```
+allprojects {
+    repositories {
+        maven {
+            url  "https://dl.bintray.com/plumeraihq/larq-compute-engine"
+        }
+    }
+}
+dependencies {
+    implementation 'org.larq:lce-lite:0.1.2
+}
+```
+
+This AAR includes binaries for 'arm64-v8a', 'x86_64', 'x86' [Android ABIs](https://developer.android.com/ndk/guides/abis).
+Please note that currently hand-optimized LCE operators are available only for 'arm64-v8a'.
+To include binaries only for Android ABIs you need to support, modify `abiFilters`
+in the Gradle build as described [here](https://www.tensorflow.org/lite/guide/android#use_the_tensorflow_lite_aar_from_jcenter).
+
+!!! note
+    In TensorFlow Lite image classification app, you need to also remove
+    the standard TensorFlow Lite dependency by commenting out the following
+    line in `build.gradle` file:
+
+    ```
+    dependencies {
+        // Comment out the following line
+        // implementation 'org.tensorflow:tensorflow-lite:0.0.0-nightly'
+    }
+    ```
+
+#### Build LCE AAR locally ####
+
+In cases that you would like to make local changes to the LCE code or
+TensorFlow Lite binaries, you might wish to build the LCE AAR locally.
+We provide a bash script to build the LCE AAR. The bash script is only
+tested inside the LCE docker container. See LCE [build guide](/compute-engine/build) to
+setup the docker container.
+
+Once the docker container is setup, run the following command from the LCE root
+directory inside the container:
+
+``` bash
+./larq_compute_engine/tflite/java/build_lce_aar.sh
+```
+The script will build the LCE Android archive 'lce-lite-\<version\>.aar` and
+store the file in the root directory of LCE repository.
+
+Transfer the LCE Adnroid archive file from the LCE container to the host
+machine (where the Android Studio is setup).
+Execute the following command to install the LCE Android Archive
+to local Maven repository:
+
+```
+mvn install:install-file \
+  -Dfile=lce-lite-0.1.aar \
+  -DgroupId=org.tensorflow \
+  -DartifactId=lce-lite -Dversion=0.1.000 -Dpackaging=aar
+```
+
+Modify the `build.gradle` file in the the Android project to
+include `mavenLocal()` repository:
+
+```
+allprojects {
+    repositories {
+        jcenter()
+        mavenLocal()
+    }
+}
+```
+
+Replace the standard TensorFlow Lite dependency with
+local LCE AAR:
+
+```
+dependencies {
+    implementation 'org.tensorflow:lce-lite:0.1.000
+}
+```
+
+### 3. Add Larq Model to the project ###
+
+In this guide, we use the Larq QuickNet model for efficient and
+fast image classification. The FlatBuffer format of the QuickNet model
+'quicknet.tflite' (see [here](/compute-engine/converter) to convert a Larq model 
+to FlatBuffer foramt)
+and its corresponding labels file `quicknet_labels.txt`
```

Comment:
Where do I obtain these labels?

Reply:
the model is trained with ImageNet. It is technically same as mobilenet with the minor difference that QuickNet ouput size is 1000 and mobilenet is 1001 (the first element is `background`) so it can be created by removing the first element in the label list of mobilenet. @lgeiger can we have a mechanism to download the labels from Larq Zoo too?

Reply:
Checkout https://docs.larq.dev/zoo/tutorials/#classify-imagenet-classes-with-bi-real-net

Reply:
I meant a way to download the labels directly from a link (I couldn't conclude from the link you posted how to do that).  I think it would be a lot easier for people who use the models for inference to have the links (especially bc the inference is not done in python)

Reply:
Not sure how mobilenet does it, but in zoo we only re-export the [decode function from keras applications](https://github.com/tensorflow/tensorflow/blob/b5085d20140eb4f492adaaefc1c307ad7ddcc197/tensorflow/python/keras/applications/imagenet_utils.py#L117-L155) which downloads the human readable labels from [this URL](https://storage.googleapis.com/download.tensorflow.org/data/imagenet_class_index.json). Does this help for the problem here?

Reply:
I just realized the image classification app also ships a `label_without_background.txt` file. so I update the the doc to use that label instead 👍